docs/contributing.rst

```diff
@@ -115,10 +122,11 @@ Example: ``./test/libsolidity/syntaxTests/double_stateVariable_declaration.sol``
         uint128 variable;
     }
     // ----
-    // DeclarationError: Identifier already declared.
+    // DeclarationError: (36-52): Identifier already declared.
 
-A syntax test must contain at least the contract under test itself, followed by the separator ``----``. The additional comments above are used to describe the
-expected compiler errors or warnings. This section can be empty in case that the contract should compile without any errors or warnings.
+A syntax test must contain at least the contract under test itself, followed by the separator ``// ----``. The following comments are used to describe the
+expected compiler errors or warnings. The number range denotes the location in the source where the error occurred.
+The section after the separator can be empty in case the contract should compile without any errors or warnings.
```

Comment:
Maybe rephrase: "In case the contract should compile without any errors or warning, the separator can be left out and the section after it can be empty" ? I'd somehow take https://github.com/ethereum/solidity/pull/5084/files#r220308437 into account. Perhaps add an example of such a contract?